packit/config/package_config.py

```diff
@@ -168,16 +168,28 @@ def find_packit_yaml(
     raise PackitConfigException("No packit config found.")
 
 
-def load_packit_yaml(config_file_path: Path) -> Dict:
+def load_packit_yaml(config_file_path: Optional[Path], raw_text: str = "") -> Dict:
```

Comment:
```suggestion
def load_packit_yaml(config_file_path: Optional[Path] = None, raw_text: str = "") -> Dict:
```

And then usage could be either `load_packit_yaml(config_file_path=something)` or `load_packit_yaml(raw_text=something)`.

Reply:
yeah, this is definitely better, I didn't want to break compatibility as I didn't know if the method is being used in packit-service. Your suggestion preserves and is a more convenient solution, thank you!